src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -205,6 +205,15 @@ public static GitHub connectAnonymously() throws IOException {
     }
 
     /**
+     * Connects to GitHub Enterprise anonymously.
```

Comment:
One wants to connect to GH Enterprise as anonymous and this was not possible until now.